modules/account/src/main/kotlin/edu/stanford/spezi/module/account/login/LoginViewModel.kt

```diff
@@ -63,6 +68,14 @@ internal class LoginViewModel @Inject constructor(
             is Action.Async.PasswordSignIn -> {
                 execute(action = action, block = ::onPasswordSignIn)
             }
+
+            is Action.EmailClicked -> {
```

Comment:
It would be great to add a unit test for this as well 🙏 . Also a pretty simply test of the login screen in account module should be enough to satisfy test coverage there. Simply launching and asserting that `LoginScreenTestIdentifier.ROOT` is displayed.